cobspec/src/test/java/com/saralein/cobspec/validation/PortValidatorTest.java

```diff
@@ -28,15 +34,16 @@ public void returnsCorrectErrorIfPortArgExcluded() {
 
     @Test
     public void returnsCorrectErrorForInvalidPortArg() {
+        String error = "Port must be a number between 1024 and 65535.";
         List<String> stringPortArgs = Arrays.asList("-p","coconuts");
         ArrayList<String> stringPortErrors = portValidator.validate(stringPortArgs);
 
-        assertTrue(stringPortErrors.contains("Port must be a number between 1 and 65535."));
+        assertTrue(stringPortErrors.contains(error));
 
         List<String> outOfRangeArgs = Arrays.asList("-p","coconuts");
```

Comment:
this is named outOfRangeArgs, but it's actually "coconuts"

Reply:
Thank you.  [Fixed that](https://github.com/saralein/java-server/pull/48/commits/02dd9356a9a57f320cd9dd9f8e3b05ff5e16f5f6#diff-2ec1fee9cbbe054ffb3bfc04395a2f9fR46).  Also, adjusted the one before it to be `nonNumericPortArg` instead of `stringPortArg`.  They are all strings at this point.